Bio/SearchIO/_model/hit.py

```diff
@@ -230,6 +236,7 @@ def __str__(self):
         return '\n'.join(lines)
 
     def __getitem__(self, idx):
+        """Return item of index idx."""
```

Comment:
Same thing here. I would phrase this as `Return the HSP object at the given index`.